pkg/apis/apis/v1alpha1/types_apiexport.go

```diff
@@ -257,6 +257,12 @@ type VirtualWorkspace struct {
 	// +kubebuilder:format:URL
 	// +required
 	URL string `json:"url"`
+
+	// Map of string keys and values that can be used to organize and categorize
+	// (scope and select) objects. May match selectors of controllers and services.
+	//
+	// +optional
+	Labels map[string]string `json:"labels,omitempty"`
```

Comment:
how is this field going to be used? could it be defined as a `string`?

Reply:
or maybe even a `struct`?

Reply:
I think that labels / selector is a common, powerful and well understood pattern. It can all be aggregated in a string but I am not sure of the benefit.